### ISSUE TITLE
How to apply IConductor approach in a Xamarin.Forms project

### ISSUE THREAD

**Issue by nologinatgit** [Caliburn-Micro/Caliburn.Micro#346]
I intend to create a Xamarin.Forms project that consists of a TabbedPage and it should have different child ContentPages (more than one).

As I read it on the wiki, it should use the IConductor approach, with a Collection.OneActive method. 

However, I can't figure out how to use it in the Xamarin.Forms world.

I can't use a ContentControl in the Xamarin Xaml (for ActiveItem), so what else should I?

Is there any example that could help me how I should use the TabbedPage view with the Conductor approach?

**Comment by nigel-sampson** [comment]
I'm currently working through this exact scenario on a new [samples branch](https://github.com/Caliburn-Micro/Caliburn.Micro/tree/samples/samples/features), it has a Conductor demo that includes a Xamarin.Forms implementation.

**Comment by nologinatgit** [comment]
Thanks for the sample.

Unfortunately, there is an issue and I need some advice.

I added to my Conductor view the following:
```
 <TabbedPage.ItemTemplate>
        <DataTemplate>
            <ContentPage Title="{Binding BaseText}" cm:View.Model="{Binding}" />
        </DataTemplate>
  </TabbedPage.ItemTemplate>
```


I created a simple test ContentPage.

These are the registration in my PCL project (I have no registration in the subproject as it caused double registrations):

 ```
 container.PerRequest<MainViewModel>();
  //container.PerRequest<TabViewModel>();
  container.PerRequest<TestViewModel>();
```

This is my conductor:

 ```
 public void AddTab()
    {      
      ActivateItem(new TestViewModel());
    }
```

I checked the events in `TestViewModel`, and the OnActivated event gets called, but the ViewAttached doesn't.

The `GetInstance` method in my Windows Phone project is called for the following types:
- SimpleContainer
- INavigationService
- MainViewModel (my conductor ViewModel)

My TestViewModel is never requested.

What am I missing?
